.github/workflows/lint.yml

```diff
@@ -0,0 +1,34 @@
+name: Lint
+
+on:
+  push:
+    branches:
+      - develop
+      - tracy
+  pull_request:
+    branches:
+      - develop
+      - tracy
+
+jobs:
+  run-linters:
+    name: Run linters 
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Check out Git repository
+        uses: actions/checkout@v2
+
+      - name: Set up Node.js
+        uses: actions/setup-node@v1
+        with:
+          node-version: "16"
+
+      - name: Install Node.js dependencies
+        run: npm ci
+
+      - name: Run linters
+        uses: wearerequired/lint-action@v2
```

Comment:
This job of Run Linters is not supposed to work like this.

change it to 

```yml
      - name: Run linters
        run: npm run lint
        with:
          eslint: true
```

I have added a script for `npm run lint` to check the files against our Linter.

So using it will be the good one